src/utils/actions.js

```diff
@@ -180,11 +180,22 @@ const actions = [
     },
     requiredInput: ['field', 'value', 'softLock'],
   },
-  // {
-  //   value: 'setVariable',
-  //   label: 'Set variable value',
-  //   group: 'variables',
-  // },
+  {
+    value: 'setVariable',
+    label: 'Set variable value',
+    group: 'variables',
+    getActionCall: ({ app, variable, value }) => async () => {
+      if (variable && value) {
+        try {
+          const variableObj = await app.getVariableByName(variable);
+          await variableObj.setStringValue(value);
+        } catch (e) {
+          // no-op
+        }
```

Comment:
Otherwise we would need to fetch all the variables and make sure it exists.
Seems to be no side effects of this

Reply:
Would it be enough to check for existence of variableObj?

Reply:
No. The error is coming from `getVariableByName` if an incorrect variable name is sent, which breaks the object :/

Reply:
Ah very strict! :)